api/v1beta1/disruption_webhook.go

```diff
@@ -245,18 +243,15 @@ func (r *Disruption) getMetricsTags() []string {
 		"namespace:" + r.Namespace,
 	}
 
-	if _, ok := r.Annotations["UserInfo"]; ok {
-		var annotation v1.UserInfo
-
-		err := json.Unmarshal([]byte(r.Annotations["UserInfo"]), &annotation)
+	if userInfo, err := r.UserInfo(); !errors.Is(err, ErrNoUserInfo) {
 		if err != nil {
-			logger.Errorw("Error decoding annotation", err)
+			logger.Errorw("error retrieving user info from disruption, using empty user info", "error", err, "name", r.Name, "namespace", r.Namespace)
 		}
 
-		tags = append(tags, "username:"+annotation.Username)
```

Comment:
I wonder if we should change the tag name to disruptionUser?

Reply:
This tag seems as of today, purely related to our metric, and I do not see any Datadog added tag resembling to this, not sure it worth to make it prefixed for this specific use case?

Looking at available data since 3 months and `username` was always filled by our values:
* https://ddstaging.datadoghq.com/metric/explorer?start=1657458970377&end=1665407770377&paused=false#N4IgNghgng9grgFxALlADxQRgDQil3AdwEsATBACwJAoFNiBzCpZTAX11NoDNiA7YgmIw+KUAigAHWihBCAtrQDOtAE7FlIXKtoBHOMoRKUAbVA6lkkSoD63GKvkQWc4opXrNufWo3HkZiB8EIqyPqpQmFogpM4QNkrwqgDGMsggigjqycbeBhGyEABuDMjJFBAwSgB0ySJZMGBgatVFEGBkzsJ8tTrOtKTAAFRsAAQARlCjwHAewYps1RBKNnVwfAgAFACUIGwAurj2jnCQ-oHH8qcQYfmRe4cgShLNYiCS7bQICGkxMAyrM5KYjJaIdPi0GwSaSyRIdUhg-iQkjkKjpPgOJxgPacYiWSBQKFSX7jCCqYwHNhsIA

We isolated `name` to `disruptionName` in metrics because `name` was an already defined tag by `host` I believe